test/helpers/transferValues.test.js

```diff
@@ -0,0 +1,13 @@
+import { assert } from 'chai'
+import { transferValues } from '../../src/helpers'
+
+describe('transferValues helper function', () => {
```

Comment:
I would like a little more tests here to better describe what this helper does.

Reply:
Do you mean that the tests in `transferValues.test.js` are not enough? What would you like to test additionally?